frontend/js/config/merge-profiles/fennica.js

```diff
@@ -92,6 +92,9 @@ import TargetRecord from '../target-record';
 import * as subrecordMergeTypes from '../subrecord-merge-types';
 import { hyphenate } from 'isbn-utils';
 
+var date = new Date().toISOString().replace(/-/g, '').replace(/T.*/, '');
```

Comment:
[Moment](http://momentjs.com/docs/) is a great module for datetime manipulation. The variable initialization can be reduced to a one-liner:
```js
const new901 = `SU${moment().format('YYYYMMDD')}`;
```

Reply:
Fixed to use Moment.